api/src/main/java/com/codeforcommunity/rest/subrouter/PublicEventsRouter.java

```diff
@@ -0,0 +1,41 @@
+package com.codeforcommunity.rest.subrouter;
+
+import static com.codeforcommunity.rest.ApiRouter.end;
+import static com.codeforcommunity.rest.RestFunctions.getMultipleQueryParams;
+
+import com.codeforcommunity.api.IPublicEventsProcessor;
+import com.codeforcommunity.dto.userEvents.responses.GetEventsResponse;
+import com.codeforcommunity.rest.IRouter;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Route;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import java.util.List;
+
+public class PublicEventsRouter implements IRouter {
+
+  private final IPublicEventsProcessor processor;
+
+  public PublicEventsRouter(IPublicEventsProcessor processor) {
+    this.processor = processor;
+  }
+
+  @Override
+  public Router initializeRouter(Vertx vertx) {
+    Router router = Router.router(vertx);
+    registerGetEvents(router);
+    return router;
+  }
+
+  private void registerGetEvents(Router router) {
+    Route getEvent = router.get("/");
+    getEvent.handler(this::handleGetEvents);
+  }
+
+  private void handleGetEvents(RoutingContext ctx) {
+    List<Integer> intIds = getMultipleQueryParams(ctx, "ids", str -> Integer.parseInt(str));
```

Comment:
```suggestion
    List<Integer> eventIds = getMultipleQueryParams(ctx, "ids", str -> Integer.parseInt(str));
```
❓ Is it logically correct to call this eventIds? Is this query param selecting which events are returned? Is it a required parameter?